frontend/packages/akr/src/pages/PrivacyPolicyPage.tsx

```diff
@@ -111,297 +117,178 @@ export const PrivacyPolicyPage = () => {
                 text="kirjaamo@oph.fi"
               />
               <br />
-              Puhelinvaihde: 029 533 1000
+              {translatePrivacy('registrar.contact.phoneSwitch')}: 029 533 1000
             </Text>
           </div>
           <div className="rows gapped">
-            <H2>Rekisterinpitäjän edustaja (yhteyshenkilö)</H2>
+            <H2>{translatePrivacy('registrarContactPerson.heading')}</H2>
             <Text>
-              Terhi Seinä
+              {translatePrivacy('registrarContactPerson.person.name')}
               <br />
-              Sähköposti:{' '}
+              {translatePrivacy('common.email')}:{' '}
               <ExtLink
                 className="privacy-policy-page__content__link"
                 href="mailto:kirjaamo@oph.fi"
                 text="kirjaamo@oph.fi"
               />
               <br />
-              Puhelin: 029 533 1000 (vaihde)
+              <Trans t={translatePrivacy} i18nKey="common.phoneSwitch">
+                029 533 1000
```

Comment:
Kun meillä taitaa olla tasan tämä yksi vaihteen puhelinnumero käytössä, niin sen voisi saman tien sisällyttää käännöstekstiin.

Reply:
Joo, voin vaihtaa näin